client/editor.cpp

```diff
@@ -183,8 +183,6 @@ void editor_ruleset_changed()
  */
 void editor_init()
 {
-  fc_assert(editor == nullptr);
```

Comment:
If the editor is not null, it's already enabled, so you don't need to create a new one, no?

Reply:
I removed because I always got an assert failure even when the was nullptr.

Reply:
That's not possible :thinking: At least not if the `editor` variable is null.

Reply:
If I am reading this right and what the `fc_assert()` does, this needs to to be `!=`. If the editor hasn't been set (def at init) then it would be not equal. Let me play with that some.